cfme/tests/containers/test_smart_management.py

```diff
@@ -42,30 +43,37 @@ def get_object_name(obj):
 
 
 def set_random_tag(instance):
+    logger.info("Setting random tag")
```

Comment:
Not very useful IMHO.

Reply:
I think this is necessary to understand where the code start failing, I can remove this but prefer to keep it that way.

What do you think?